qutebrowser/config/configdata.py

```diff
@@ -1466,6 +1466,9 @@ def data(readonly=False):
         ('set-cmd-text :open -b -i {url:pretty}', ['xO']),
         ('set-cmd-text -s :open -w', ['wo']),
         ('set-cmd-text :open -w {url:pretty}', ['wO']),
+        ('set-cmd-text -s :search', ['/']),
+        ('set-cmd-text -s :search --r', ['?']),
```

Comment:
Sorry, missed that with the last review: This should be `-r` and not `--r` (single letter options have single dashes)